internal/parser/planparserv2/Plan.g4

```diff
@@ -74,6 +76,9 @@ BXOR: '^';
 AND: '&&' | 'and';
 OR: '||' | 'or';
 
+ISNULL: 'is_null' | 'IS_NULL';
```

Comment:
Do we need to support grammar like `col IS NULL`, instead of `IS_NULL(col)`?

Reply:
it's hard to parse col name if col is ahead of ISNULL

Reply:
and I think IS_NULL(col) format is enough for users.